packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -3341,7 +3348,7 @@ class _TimePickerDefaultsM2 extends _TimePickerDefaults {
       // TODO(rami-a): Remove this workaround once
       // https://github.com/flutter/flutter/issues/54104
       // is fixed.
-      errorStyle: const TextStyle(fontSize: 0, height: 0),
+      errorStyle: const TextStyle(fontSize: 0.0, height: 1.0),
```

Comment:
The ".0" part isn't needed, the compiler will know it's a `double`. It used to be required or it would think it was an `int`, but that was years ago.

Reply:
Thanks for letting me know!